js/views/modals/wallet/Wallet.js

```diff
@@ -80,7 +80,7 @@ export default class extends BaseModal {
       active: coin === opts.initialNavCoin,
       code: coin,
       name: app.polyglot.t(`cryptoCurrencies.${coin}`, { _: coin }),
-      balance: coin.confirmed,
+      balance: app.walletBalances.get(coin).get('confirmed'),
```

Comment:
Would you mind checking first if the balance model is there? It should be, but the server has given us quirkier things and I'm trying to more defensively program, at least on the low hangin fruit.

```
const balanceMd = app.walletBalances.get(coin);

...

balance: balanceMd && balanceMd.get('confirmed'),
```